Dockerfile

```diff
@@ -300,6 +300,22 @@ ENV PATH "/opt/buildhome/.yarn/bin:$PATH"
 
 USER root
 
+################################################################################
+#
+# Deno
+#
+################################################################################
+
+RUN if [ "$TARGETARCH" = "amd64" ] ; then curl -o- -L https://deno.land/x/install/install.sh > /usr/local/bin/deno-installer.sh; fi
+ENV DENO_VERSION=v1.25.4
```

Comment:
Might be good to add a comment explaining that this env variable will be used by the `install.sh` script to determine which version to download?

Reply:
the line where it's used is two lines down, not sure if a comment is necessary. we could make this an `ARG` instead of `ENV` to make this more obvious

Reply:
`install.sh` itself contains no reference to `DENO_VERSION`, only to `$1`

Reply:
Oh, sorry I totally misread this. Ignore me!